FrOST_Autogen_Config.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+# Install WSL Packages
+echo "Updating and Installing Packages..."
+sudo apt update;
+sudo apt install gcc-arm-linux-gnueabihf -y;
+sudo apt install wsl -y;
+sudo apt install dos2unix -y;
+sudo apt install libssl-dev -y;
+
+# Install the Device Tree Compiler
+echo "Installing the Device Tree Compiler."
+./docs/scripts/dtc_installer.sh # This will fail to build a Python 2 package but that doesn't matter
+
+# Clone the FrOST Repos
+echo "Cloning the FrOST Git Repos."
+./docs/scripts/clone_Frost_Repos.sh
+
+# Clone and Configure Linux-SOCFPGA
+cd ~; echo "Moving to Home Directory of WSL."
+#git clone https://github.com/fpga-open-speech-tools/linux-socfpga.git;
```

Comment:
Uncomment and check for the repo already existing
```suggestion
[ ! -d 'linux-socfpga' ] && git clone https://github.com/fpga-open-speech-tools/linux-socfpga.git;
```

Reply:
Sounds good. @fe-tdavis had the linux repo clone commented out for testing.